conf/messages.en

```diff
@@ -332,16 +332,24 @@ confirmation.print.afterLink = to make a record of your submission.
 confirmation.whatNext.h2 = What happens next
 confirmation.whatNext.p1 = The movement will be updated to show you have successfully submitted an explanation for shortage or excess. This may not be happen straight away.
 confirmation.mayNeedToDo.h2 = What you may need to do
-confirmation.mayNeedToDo.p1 = This explanation for a shortage or excess has been submitted to HMRC only. If you are a Northern Ireland warehousekeeper receiving goods from an EU member state, this information will also be visible to the state of dispatch.
+confirmation.mayNeedToDo.p1.dutypaid = This explanation for a shortage or excess has been submitted to HMRC only. If you are a Northern Ireland certified or temporary certified consignee receiving goods from an EU member state, this information will also be visible to the authorities in the state of dispatch.
+confirmation.mayNeedToDo.p1.dutysuspended = This explanation for a shortage or excess has been submitted to HMRC only. If you are a Northern Ireland warehousekeeper receiving goods from an EU member state, the information will also be visible to the authorities in the state of dispatch.
 confirmation.mayNeedToDo.shortage.h3 = If you recorded a shortage
-confirmation.mayNeedToDo.shortage.p1 = A duty point may be created for any goods in this movement that are short. This means HMRC will require duty to be paid on those goods by the person or organisation that is holding them.
+confirmation.mayNeedToDo.shortage.p1.dutypaid = You may be able to reclaim overpaid duty on the goods in this movement that are short.
+confirmation.mayNeedToDo.shortage.p2.dutypaid = For information on what to do next if you discover shortages, see
+confirmation.mayNeedToDo.shortage.p2.dutypaid.urltext = [Section 6.5 of Excise Notice 204b].
```

Comment:
Not sure this needs square brackets.

Reply:
Pushed a change to remove the square brackets